packit.yaml

```diff
@@ -7,6 +7,7 @@ specfile_path: cockpit-files.spec
 copy_upstream_release_description: true
 
 srpm_build_deps:
+  - jq
```

Comment:
You said "js" in the commit message.

I wonder if we should maybe port that node-modules script to Python at some point... it was written back before we were so thoroughly tied to Python in Cockpit... but that's a job for another day...